cypress/integration/version/downstream_tasks.ts

```diff
@@ -14,15 +14,17 @@ describe("Downstream Tasks Tab", () => {
     cy.visit(DOWNSTREAM_TASKS_ROUTE);
 
     cy.dataCy("project-accordion").should("have.length", 3);
-
+    cy.wait(0);
```

Comment:
Added this line as a recommendation from https://github.com/cypress-io/cypress/issues/7306
`project-accordion` on line 18 was detaching from the between the time .first() and .click() were executed.

Reply:
Can we add a comment here with a description of why its necessary.